readme.md

```diff
@@ -77,3 +76,41 @@ The docker containers and images created by the tests framework are removed when
 The docker images and containers have to be removed manually if CLEAN_UP_TEST_ENVIRONMENT=false. The Pyrsia integration
 tests also create the temp directory `/tmp/pyrsia_tests`which is not removed by the tests framework and if necessary has to be removed
 manually.
+
+## Troubleshooting
+
+In case of any problems with the tests environment reset the enviroment as follow:
+1) Remove all docker images and containers:
+   ```sh
+      docker system prune --all
+   ```
+2) Remove the integration tests temp directory:
+   ```sh
+      rm -rf /tmp/pyrsia_tests/
+   ```
+
+## Tests logger
+
+Supported logging levels:
+- INFO (default)
+- DEBUG
+- ERROR
+
+How to use logger in the tests:
+
+```sh
+  # load the test from the library
+  load '../lib/logger/load'
+  
+  # print logging messages
```

Comment:
It may be helpful to give examples using variables or commands.

```
log DEBUG "Debug test message! $PYRSIA_CLI"
log ERROR "Error test message! ${$PYRSIA_CLI ping}"
```

Reply:
Maybe but I think the pattern of using vars in strings is pretty common and most people should be familiar with it.

Reply:
Yeah I wondered if it may be too basic. Thanks.